AdobeStockImageAdminUi/Ui/Component/Listing/Filter/Color.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\AdobeStockImageAdminUi\Ui\Component\Listing\Filter;
+
+use Magento\Framework\Api\FilterBuilder;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Filters\FilterModifier;
+use Magento\Ui\Component\Form\Element\Input as ElementInput;
+use Magento\Ui\Component\Filters\Type\AbstractFilter;
+use Magento\Ui\Model\ColorPicker\ColorModesProvider;
+
+class Color extends AbstractFilter
+{
+    const NAME = 'filter_input';
```

Comment:
why do you add their const for one-time usage? Usually, we have const only interfaces`